src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -269,6 +269,14 @@ export type UI = {
     extended?: boolean | undefined,
   ) => void;
   readonly close: () => void;
+  readonly setPosition: (options: SetUIPositionOptions) => void;
```

Comment:
- Only myself can be moved. Other widget cannot be moved.
- Renamed to moveTo, because it generates side effects (reloading plug-ins) that cannot be imagined from the light-impression of the word "override".
- I thinks it is better to move under `Widget`.

```js
reearth.widget.moveTo(WidgetPositonOptions)
```

Reply:
Updated:
- `reearth.widget.moveTo(WidgetLocationOptions)`
- Internal functionality naming as `moveWidget`
- For the `alignSystem` i still use `overriddenAlignSystem` & `setOverrideAlignSystem` because `moveWidget` is actually modifying the `alignSystem` object.